compiler/include/symbol.h

```diff
@@ -116,6 +116,7 @@ enum ForallIntentTag {
   TFI_REDUCE,                      //   AS    (for Accumulation State)
   TFI_TASK_PRIVATE,                //   TPV
   // compiler-added helpers
+  TFI_COMPILER_ADDED_START,        //   (never used)
```

Comment:
IMO it would have been a smaller change to just add a function to tell you if a ForallIntentTag is compiler added or user specified. However I am OK with this as-is.